pom.xml

```diff
@@ -309,8 +309,8 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-compiler-plugin</artifactId>
         <configuration>
-          <source>1.7</source>
-          <target>1.7</target>
+          <source>1.8</source>
+          <target>1.8</target>
```

Comment:
Argh... Checked in a private change. Mockito won't run tests unless compiled with JDK 7. Eclipse won't run tests unless I recompile for JDK 8. Reverted.